packages/fast-data-utilities-react/src/index.ts

```diff
@@ -373,6 +516,58 @@ export function mapDataToComponent(
     // organize by length using split "."
     reactChildrenDataLocations.sort(orderChildrenByDataLocation);
 
+    // find locations of all items of data that are overridden by mappings
+    const pluginModifiedDataLocations: PluginLocation[] = getDataLocationsOfPlugins(
+        schema,
+        mappedData,
+        childOptions
+    );
+
+    // remove any children data locations from plugin modified locations
+    reactChildrenDataLocations = reactChildrenDataLocations.map(
+        (reactChildrenDataLocation: string): string => {
+            if (
```

Comment:
this doesn't always return a string - is that a problem? If not, can you update the type signature?